src/bake/bake.private.d.ts

```diff
@@ -40,6 +48,7 @@ declare type EncodedDependencyArray = (string | number)[];
 declare type UnloadedCommonJS = (
   hmr: import("./hmr-module").HMRModule,
   module: import("./hmr-module").HMRModule["cjs"],
+  exports: any,
```

Comment:
consider `unknown`

Reply:
i did but unknown in general for these codebases just leads to using `as any` which doesn't actually help anyone out.

Reply:
though, in this case i can do that since i never define this function, only calling it